analyses/molecular-subtyping-LGAT/run_subtyping.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+# Run notebook to get molecular subtype for LGAT samples 
+
```

Comment:
You should add `00-subset-files-for-LGAT.R` here but within some logic for CI - take a look at this file for an example: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/d197f2717eb0d067fe35644db91c4910b6d46c06/analyses/molecular-subtyping-ATRT/run-molecular-subtyping-ATRT.sh

Reply:
Also will generate a new subset file when folks run this locally (see the note [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-ATRT#molecular-subtyping-atrt))